pagopa/io-app
# feat(IT Wallet): [SIW-212] Add pid issuing identification screen

## Short description
This PR adds the identification screen when the user wants to add issued credentials to its wallet.

## List of changes proposed in this pull request
- Update credentials.ts
- Update itwCredentials reducer

## How to test
Start app and try to activate the wallet. Tap to add issued pid and the identification screen should appear.
